### ISSUE TITLE
has function throws error when there is no query object

### ISSUE THREAD

**Issue by JaZo** [fernandofleury/query-object#12]
If you use queryObject.has('foo') when there is no query object, an error is thrown. I would expect it to return false in this case.
## Example

Without query object

``` javascript
// example.com

queryObject.has('bar');
// query-object.js:177 Uncaught TypeError: Cannot read property 'hasOwnProperty' of undefined
```

With query object

``` javascript
// example.com?foo

queryObject.has('bar');
// returns false
```

**Comment by fernandofleury** [comment]
@JaZo Will work on it. Thanks for the feedback

**Comment by fernandofleury** [comment]
@JaZo Fixed on [2.0.3](https://github.com/fernandofleury/query-object/releases/tag/v2.0.3)
